class/frontend-account.php

```diff
@@ -411,4 +412,40 @@ public function update_profile() {
     public function get_allowed_cpt() {
         return wpuf_get_option( 'cp_on_acc_page', 'wpuf_my_account', [ 'post' ] );
     }
+
+    /**
+     * Check if the user is the post author and give permission for previewing
+     *
+     * @since WPUF
+     *
+     * @return void
+     */
+    public function process_user_for_previewing_post( $query ) {
```

Comment:
Better handle admin. admin previewing posts often.
`if ( is_admin() ) return;`

Reply:
> Better handle admin. admin previewing posts often. `if ( is_admin() ) return;`

Bhai as discussed with you, current_user_can( 'edit_posts' ) will check for the admin and other users who has the 'edit_posts' capability